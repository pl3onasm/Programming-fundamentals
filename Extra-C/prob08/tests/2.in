6:[1,5),[7,8),[8,10),[12,15),[2,3),[4,6)
