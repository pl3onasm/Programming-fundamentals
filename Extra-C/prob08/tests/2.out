[1,6),[7,10),[12,15)
